apps/climatemappedafrica/src/lib/data/blockify/hero.js

```diff
@@ -37,7 +38,7 @@ export default async function hero(block, _api, _context, { hurumap }) {
     ...block,
     boundary,
     center,
-    explorePageSlug,
+    explorePageSlug: enableHURUMap ? explorePageSlug : null,
```

Comment:
1. It's `hurumap.enabled` and not `hurumap.enableHURUMap`.
2. Looking at the relevant code [here](https://github.com/CodeForAfrica/ui/blob/008da14a7781af4fffa59861c1221c27c61e1ed5/apps/climatemappedafrica/src/lib/data/common/index.js#L119-L123), if `hurumap` is not `enabled`, `profilePage` would be `undefined` i.e. no need to check for `hurumap.enabled`, just check `profilePage?.slug || null`.